src/basics.php

```diff
@@ -143,3 +143,26 @@ function json_last_error_msg()
     }
 
 }
+
+if (!function_exists('breakpoint')) {
+    /**
+     * Command to return the eval-able code to startup PsySH in interactive debugger
+     * Works the same way as eval(\Psy\sh());
+     * psy/psysh must be loaded in your project
+     * @link http://psysh.org/
+     * ```
+     * eval(breakpoint());
+     * ```
+     * @return string
+     */
+    function breakpoint()
+    {
+        if (PHP_SAPI === 'cli' && class_exists('\Psy\Shell')) {
+            return 'extract(\Psy\Shell::debug(get_defined_vars(), isset($this) ? $this : null));';
+        }
+        trigger_error(
+            "psy/psysh must be loaded and you should be in CLI to use the breakpoint method",
```

Comment:
"psy/psysh must be installed and you must be in a CLI environment to use the breakpoint function" possibly?